pkg/extender/extender.go

```diff
@@ -273,14 +273,25 @@ func (e *Extender) getNodeScore(
 	regionInfo *localityInfo,
 	idMap map[string]*volume.NodeInfo,
 ) int {
+	logger := log.WithFields(log.Fields{
+		"node": node.Name,
+	})
+	logger.Debugf("getNodeScore, let's go")
 	for _, address := range node.Status.Addresses {
 		if address.Type != v1.NodeHostName {
 			continue
 		}
+		logger.Debugf("rack info: %+v", rackInfo)
+		logger.Debugf("zone info: %+v", zoneInfo)
+		logger.Debugf("region info: %+v", regionInfo)
```

Comment:
This isn't required. It is already printed in `prioritizeNodes` and doesn't change in each iteration of the loop